core/explorer.go

```diff
@@ -28,10 +37,11 @@ type ExplorerClient struct {
 }
 
 type ExplorerDID struct {
-	PeerID    string `json:"peerid"`
+	PeerID    string `json:"peer_id"`
 	DID       string `json:"user_did"`
-	IPAddress string `json:"ipaddress"`
+	IPAddress string `json:"ip_addess"`
```

Comment:
IPAddress is not needed

Reply:
Yes, removed it. When it was still there, I was getting error that its mandatory field